meetings/cpansec-minutes-2024-10-02.md

```diff
@@ -0,0 +1,112 @@
+## Meeting details
+
+* 2024-10-02 17:00 UTC on #cpansec-discussion on Matrix
+
+## 16:30 UTC – Pre-meeting socializing
+
+*   Discuss organizing projects, swimlanes and issues (...)
+
+## 17:00 UTC – Meeting start
+
+### Welcome
+
+*   Meeting chair: @timlegge
+*   Meeting secretary: @tux & #team
+
+### Attendees, absents & regrets
+
+*   Attendees
+    * @sjn
+    * @stigtsp
+    * @tux
+    * @timlegge
+*   Partly Absent
+    * @garu
+
+### Approve previous meeting minutes
+
+*   Previous meeting minutes was approved by @sjn, @timlegge, & @stigtsp
+
+### Quick summary of current work
+
+1.  [CPAN Metadata & Software Bills of Materials](https://github.com/orgs/CPAN-Security/projects/1)
+    *   …
+2.  [CPAN Privacy and Compliance](https://github.com/orgs/CPAN-Security/projects/9)
+    *   …
+3.  [CPAN Provenance & Supply Chain Security](https://github.com/orgs/CPAN-Security/projects/3)
+    *   …
+4.  [CPAN Security Outreach & Information](https://github.com/orgs/CPAN-Security/projects/12)
+    *   …
+5.  [CPAN Security Patch Tooling](https://github.com/orgs/CPAN-Security/projects/11)
+    *   …
+6.  [CPAN Software Composition Analysis](https://github.com/orgs/CPAN-Security/projects/6)
+    *   …
+7.  [CPAN Transparency Logs](https://github.com/orgs/CPAN-Security/projects/2)
+    *   …
+8.  [CPAN Vulnerability Index](https://github.com/orgs/CPAN-Security/projects/10)
+    *   …
+9.  [CPANSec Governance, Policy & Funding](https://github.com/orgs/CPAN-Security/projects/7)
+    *   …
+
+### TLS/HTTPS in core
+*   Neither @BooK nor @leont present, so no updates
+
+### German Sovereign Tech fund is open for applications
+*   @sjn - stalled: work is registered in a spreadsheet
+*   @sjn - had a chat with STF at #OSSummit
+*   @sjn summarizes results - a company involment would smooth things. Ovid said he could help
+
+### Ongoing vulnerabilities
+*   @tux checked with p5p. No further action required
+*   Someone should have a look at https://github.com/mojolicious/mojo/pull/1791 and consider if this is a vulnerability
```

Comment:
Who is "Someone"? (proposal – someone who cares & is capable. @stigtsp?)

General principle: Minutes that don't say something about a person doing or going to do something, are bad minutes. New factual information is good too!

Reply:
I prefer not to evaluate this since I'm also doing a PR to address it.

Reply:
Not sure if I'm competent and/or have enough tuits to do this. :disappointed: 

Maybe @garu ? :smile_cat:

Reply:
Sure, I volunteer. But that should not interfere with this merge, since _at the meeting_ it was just raised that someone should (and volunteering happened as it was raised outside the meeting itself).